std/math.d

```diff
@@ -5252,7 +5253,7 @@ public:
     assert(ieeeFlags == f);
 }
 
-version(D_HardFloat) @safe unittest
```

Comment:
I versioned this out sometime back, but looking around it appears that some soft-float implementatons do support checking these IEEE flags. I guess that's why there's two different structs `IeeeFlags` and `FloatingPointControl`, the former possibly supporting software floating-point implementations, while the latter exclusively for hardware? If so, this test block should be run on soft-float too.